src/nplinker/loader.py

```diff
@@ -315,15 +311,15 @@ def _init_paths(self):
             self.OR_MOLNETENHANCER) or os.path.join(self._root,
                                                     'molnetenhancer')
     def _start_downloads(self):
-        self._root = self._downloader.project_file_cache
-        logger.debug('remote loading mode, configuring root={}'.format(
-                self._root))
-            # CG: to download both MET and GEN data
-        self._downloader.get(
-                self._docker.get('run_bigscape', self.RUN_BIGSCAPE_DEFAULT),
-                self._docker.get('extra_bigscape_parameters',
-                                 self.EXTRA_BIGSCAPE_PARAMS_DEFAULT),
-                self._use_mibig, self._mibig_version)
+        downloader = PODPDownloader(self._platform_id)
```

Comment:
Why `downloader` is no more an internal attribute (before it was `_downloader`)?

Reply:
the `self._downloader` was only used in this `_start_downloads` method, so it's better to make it as an local variable.